tests/run/extension-override.scala

```diff
@@ -0,0 +1,12 @@
+class A:
+  extension (s: String)
+    def len: Int = s.length
+
+object B extends A:
+  extension (s: String)
+    override def len: Int = s.length + 1
+
+@main def Test =
+  import B._
+  assert("abc".len == 4)
+
```

Comment:
I'm not sure how much of an issue this should be, but this import suggestions error masks the fact that the real error is that `B.len` is an illegal override, as we can see I am importing `B.len` in Test
```scala
scala> class A:                                                                                                         
     |   extension (s: String)
     |     def len: Int = s.length
     | 
     | object B extends A:
     |   def len(s: String): Int = s.length + 1
     | 
     | @main def Test =
     |    import B.len
     |    assert("abc".len == 4)
     | 
10 |   assert("abc".len == 4)
   |          ^^^^^^^^^
   |value len is not a member of String, but could be made available as an extension method.
   |
   |The following import might fix the problem:
   |
   |  import B.len
   |
```

Reply:
I think that could be a possible improvement to the import suggestions. I.e. don't make a suggestion if the name is already in scope. Maybe open an issue?

Reply:
opened https://github.com/lampepfl/dotty/issues/10231